### ISSUE TITLE
Move TODOs from code to issue tracker

### ISSUE THREAD

**Issue by dhermes** [google/apitools#3]
@craigcitro WDYT of this?

------

To give some context, I noticed the large number of TODO comments after filing #2, which had an old link to https://github.com/craigcitro/apitools, so I did a `git grep -i citro`.

**Comment by craigcitro** [comment]
yeah, this is a good idea. i see 90 TODOs on the internal code, so i should probably skim through and delete some first.

is there an easy tool to convert TODOs to issues? the closest thing i know is this: https://github.com/google/todo-tracks

**Comment by dhermes** [comment]
Easiest tool: hire an intern :)

**Comment by cherba** [comment]
Looks like number of TODOs have come down to 25
https://github.com/google/apitools/search?utf8=%E2%9C%93&q=TODO
